Makefile

```diff
@@ -112,9 +119,7 @@ check: dependencies
 	@echo "Checking for incorrect conditional trait interface overrides..."
 	@mono --debug OpenRA.Utility.exe all --check-conditional-trait-interface-overrides
 
-
-NUNIT_CONSOLE := $(shell test -f thirdparty/download/nunit3-console.exe && echo mono thirdparty/download/nunit3-console.exe || \
-	which nunit3-console 2>/dev/null || which nunit2-console 2>/dev/null || which nunit-console 2>/dev/null)
+NUNIT_CONSOLE := $(which nunit3-console 2>/dev/null || which nunit2-console 2>/dev/null || which nunit-console 2>/dev/null)
```

Comment:
Does `nunit2` even support our new minimum .net version? This may be a good time to drop support and references to it.

Reply:
```
NUNIT_CONSOLE := $(shell test -f ~/.nuget/packages/nunit.consolerunner/3.11.1/tools/nunit3-console.exe && echo mono ~/.nuget/packages/nunit.consolerunner/3.11.1/tools/nunit3-console.exe || \
	which nunit3-console 2>/dev/null || which nunit2-console 2>/dev/null || which nunit-console 2>/dev/null)
```
Do we need to have the `which` part if we only going to support nunit3?

Reply:
Could prolly remove all of this and just use `"mono ~/.nuget/packages/nunit.consolerunner/3.11.1/tools/nunit3-console.exe --noresult OpenRA.Test.nunit.dll"`

Reply:
IMO it would probably be simpler if we removed this completely from the makefile, hardcoded that mono line into the travis script, and required people who want to run the tests manually to use their IDE.